soc/xtensa/intel_adsp/ace/multiprocessing.c

```diff
@@ -7,6 +7,8 @@
 #include <zephyr/kernel.h>
 #include <zephyr/sys/check.h>
 #include <zephyr/arch/cpu.h>
+#include <zephyr/pm/pm.h>
+#include <ksched.h>
```

Comment:
why are you including `ksched.h` here ?

Reply:
Thanks for catching this. Those are leftovers from the previous version. I forgot to delete while editing.